bin/otomi

```diff
@@ -211,6 +210,7 @@ function drun() {
   # - in docker AND force docker
   if [ "$has_docker" = 'true' ] && [ -z "$IN_DOCKER" ]; then
     [ -n "$VERBOSE" ] && echo "Running dockerized version of command: $command"
+    [ "$image_tag" = 'latest' ] || [ "$image_tag" = 'master' ] && [ "$has_docker" = 'true' ] && docker pull "otomi/core:$image_tag"
```

Comment:
Thank you!

Reply:
I see it checks for $has_docker two times? (line 212 & 214).

Reply:
you may refactor it in your next delivery ;)

Reply:
#392